db/sql/queries/network_os_vulnerability.sql

```diff
@@ -38,6 +38,52 @@ INNER JOIN operating_systems os
   ON nov.operating_system_id = os.id
 WHERE nov.scan_id = $1;
 
+-- name: GetNetworkOSVulnerabilityWithServiceDetailByScanID :many
+SELECT
+  nov.vulnerability_id,
+  nov.scan_id,
+  nov.host_id,
+  nov.service_id,
+  nov.created_at AS vuln_created_at, 
+  nov.updated_at AS vuln_updated_at,
+  se.sv_name,
+  se.sv_version,
+  se.confidence,
+  se.cpe,
+  se.product,
+  se.protocol,
+  se.port,
+  se.port_state,
+  se.created_at as sv_created_at,
+  se.updated_at as sv_updated_at
+FROM network_os_vulnerabilities nov
+INNER JOIN services se
+  ON nov.service_id  = se.id
+WHERE nov.scan_id = $1;
+
+-- name: GetNetworkOSVulnerabilityWithServiceDetailByScanAndServiceID :many
+SELECT
+  nov.vulnerability_id,
+  nov.scan_id,
+  nov.host_id,
+  nov.service_id,
+  nov.created_at AS vuln_created_at, 
+  nov.updated_at AS vuln_updated_at,
+  se.sv_name,
+  se.sv_version,
+  se.confidence,
+  se.cpe,
+  se.product,
+  se.protocol,
+  se.port,
+  se.port_state,
+  se.created_at as sv_created_at,
+  se.updated_at as sv_updated_at
+FROM network_os_vulnerabilities nov
+INNER JOIN services se
```

Comment:
La misma observacion aqui: 


```suggestion
LEFT JOIN services se
```

Reply:
Fixed